### ISSUE TITLE
reason for 2 separate classes, validation dataset

### ISSUE THREAD

**Issue by tomek-l** [tobybreckon/fire-detection-cnn#43]
I really liked reading your paper! It communicates everything very clearly and was a joy to read. The findings will be definitely very useful in my own research. I have a few questions I'd like to ask.

Q1: Given that classes 'fire' and 'no fire' are mutually exclusive; what is the rationale for using two separate outputs? Does it have some benefits over having a single output [0-1] and thresholding it appropriately?

Q2: Could you tell me how do you sample the validation set? I found that the usual technique of randomly sampling 10-20% of the training set doesn't really work for me. Such a random subset of the training set contains images very similar to the ones remaining in the training set and isn't a good measure of out-of-sample error. 

Q3: (This might also help answer Q2). My understanding is that you don't use the entire dataset of 365,702 images, but only a subset of 23,408 images. Could you elaborate how do you pick the 6.4% of images you use?

**Comment by tobybreckon** [comment]
Answers as follows:

A1: The follows the use of softmax() and cross-entropy loss in the final year, as opposed to a sigmoid output (for a single output). This follows the highly successful use of this within the "parent"  network architectures (AlexNet + InceptionV1) from which our work is drawn.

A2: See the paper - search for word "validation", full detail clearly provided.

A3: randomly selected subset; in order to have a reasonable training time

Hope this helps.
